serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service-with-kn-cli-and-vscode.adoc

```diff
@@ -17,7 +17,7 @@ minikube start --cpus 4 --memory 8096 --addons registry --addons metrics-server
 ----
 * (optional) Install https://k9scli.io/[k9scli.io] 
 * xref:cloud/operator/install-serverless-operator.adoc[]
-* Install https://github.com/kiegroup/kie-tools/releases/tag/0.30.3[KN Workflow CLI] by downloading the correct distribution for your development environment and adding it to the PATH.
+* Install https://github.com/kiegroup/kie-tools/releases/tag/0.32.0[SonataFlow plug-in for Knative CLI]. Follow xref:testing-and-troubleshooting/kn-plugin-workflow-overview.adoc[] guide.
```

Comment:
To move this version to a variable in the `antora.yaml`

Reply:
Done